### PR TITLE
docs: Documentation improvements and typo fixes

### PR DESCRIPTION
**Pull request by exequiel09**

### CODE REVIEW

File: src/collections/_documentation/learn/cli/installation.md [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
 
 ```bash
 curl -sL https://sentry.io/get-cli/ | bash
 ```
 
-This will automatically download the correct version of `sentry-cli` for your operating system and install it. If necessarily it will prompt for your admin password for `sudo`.
+This will automatically download the correct version of `sentry-cli` for your operating system and install it. If necessarily, it will prompt for your admin password for `sudo`.
 
-To verify it’s installed correctly you can bring up the help:
+To verify if it’s installed correctly, you can bring up the help:
```
Comment (untitaker) [inline src/collections/_documentation/learn/cli/installation.md]: This is already correct english

File: src/collections/_documentation/learn/breadcrumbs.md [hunk 2]
```diff
@@ -48,15 +48,15 @@ Type
 
 ## Recording Crumbs
 
-Manual breadcrumb recording is also available and easy to use.  This way breadcrumbs
-can be added whenever something interesting happens.  For instance it might make sense
+Manual breadcrumb recording is also available and easy to use.  In this way, breadcrumbs
```
Comment (untitaker) [inline src/collections/_documentation/learn/breadcrumbs.md]: This was already correct

Comment (exequiel09) [reply src/collections/_documentation/learn/breadcrumbs.md]: will push a new commit to correct it again 👍

**Comment by untitaker** [comment]
Thanks!

**Comment by exequiel09** [comment]
You're welcome 💯

**Pull request merged**